scripts/generate/kube-events-exporter-non-rbac.jsonnet

```diff
@@ -0,0 +1,18 @@
+local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
```

Comment:
I don't think we should continue to encourage setups without rbac, I think we should just skip this